content/contributing-code/contents.lr

```diff
@@ -17,10 +17,12 @@ Here's a list of [all our current projects](/projects). We use GitHub issues ass
 We make extensive use of issue labels to desginate the status of various issues. We have a standard set of labels across all projects, [documented here](https://github.com/creativecommons/ccos-scripts/blob/master/normalize_repos/labels.py). Here are some of the ones that are most relevant to finding a good issue to work on:
 
 * Issues tagged `help wanted` and/or `good first issue` are available for community contribution. You may work on an issue labeled `good first issue` even if it's not your first issue.
-* Issues tagged `in progress` are being worked on by someone else. If there hasn't been any issue or PR activity for at least 7 days, issues with this tag can be reassigned to someone else.
-* Issues tagged `blocked`, `not ready for work` or `ticket work required` are not yet ready for work. They need something else to be done first before they are ready.
-* Issues tagged `CC staff only` are not suitable for community contribution, usually because it requires infrastructure access or institutional knowledge that would be impractical to provide to the community.
-* Issues without any of the above labels may be open for contribution.
+    * Check the issue comments to see whether someone else has indicated that they are working on it. If there hasn't been any issue or PR activity for at least 7 days, then that issue can be considered up for grabs.
+* Issues tagged `in progress` are being worked on by someone else.
+    * If there hasn't been any issue or PR activity for at least 7 days, then that issue can be considered up for grabs.
+* Issues tagged `blocked`, `not ready for work` or `ticket work required` are not yet ready for work. They need something else to be done first before they are ready. Do not work on these issues.
+* Issues tagged `CC staff only` are not suitable for community contribution, usually because it requires infrastructure access or institutional knowledge that would be impractical to provide to the community. Do not work on these issues.
+* Issues without any of the above labels _may_ be open for contribution. Please add a comment to ask whether the issue is available.
```

Comment:
Do we be explicit saying the `Good First Issue` and `Help wanted` don't need to ask for permission to start working?